File: 215-kth-largest-element-in-an-array/215-kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        int l=2*(int)Math.pow(10,4)+1;
        int s=0;
        int freq[]=new int[l+1];
        for(int i=0;i<=l;i++)
            freq[i]=0;
        for(int i=0;i<nums.length;i++)
            freq[nums[i]+(int)Math.pow(10,4)]+=1;
        for(int i=l-1;i>=0;i--)
        {
            if (freq[i]!=0)
            {
                s+=freq[i];
                if(s>=k)
                    return (int)(i-(int)Math.pow(10,4));
            }
            else
                continue;
        }
        return 0;
    }
}

File: 215-kth-largest-element-in-an-array/215-kth-largest-element-in-an-array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        freq=[0]*(2*(10**4)+1)
        s=0
        for i in nums:
            freq[i+10**4]+=1
        for i in range(len(freq)-1,0,-1):
            if freq[i]!=0:
                s+=freq[i]
                if s>=k:
                    return (i-10**4)
            else:
                continue
            
                
        